From 0ed1e393b3f1e48cb9b00eebb8e96eb14c8b3012 Mon Sep 17 00:00:00 2001
From: John Greene <jogreene@redhat.com>
Date: Mon, 8 Sep 2014 15:07:47 -0400
Subject: [ethernet] ixgbevf: delete unneeded call to pci_set_power_state

Message-id: <1410188883-6595-33-git-send-email-jogreene@redhat.com>
Patchwork-id: 92379
O-Subject: [RHEL7.1 PATCH 32/48] ixgbevf: delete unneeded call to pci_set_power_state
Bugzilla: 1091124
RH-Acked-by: Neil Horman <nhorman@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: John Linville <linville@redhat.com>

BZ: 1091124
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=7919168
Tested: In beaker. See BZ for details

This driver does not need to adjust the power state on suspend, so the
call to pci_set_power_state in the resume function is a no-op.  Drop it,
to make the code more understandable.

Signed-off-by: Julia Lawall <Julia.Lawall@lip6.fr>
Tested-by: Phil Schmitt <phillip.j.schmitt@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
(cherry picked from commit 2f586f6bcd5367fbbd1d3352d524a3ef3183eeb2)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/ixgbevf/ixgbevf_main.c b/drivers/net/ethernet/intel/ixgbevf/ixgbevf_main.c
index 85ee750..3ee8148 100644
--- a/drivers/net/ethernet/intel/ixgbevf/ixgbevf_main.c
+++ b/drivers/net/ethernet/intel/ixgbevf/ixgbevf_main.c
@@ -3305,7 +3305,6 @@ static int ixgbevf_resume(struct pci_dev *pdev)
  struct ixgbevf_adapter *adapter = netdev_priv(netdev);
  u32 err;
 
- pci_set_power_state(pdev, PCI_D0);
  pci_restore_state(pdev);
  /*
   * pci_restore_state clears dev->state_saved so call
-- 
1.7.1